CMakeLists.txt

```diff
@@ -143,7 +143,11 @@ if(AMR_WIND_ENABLE_ASCENT)
   if (Ascent_FOUND)
     message(STATUS "Found Ascent = ${ASCENT_INCLUDE_DIRS}")
   endif()
-  target_link_libraries(${amr_wind_lib_name} PUBLIC ascent::ascent)
+  if (AMR_WIND_ENABLE_MPI)
+    target_link_libraries(${amr_wind_lib_name} PUBLIC ascent::ascent_mpi)
```

Comment:
Does `Ascent` provide `COMPONENTS` so that the `find_package` in line 142 can check for `MPI` component?